core/src/main/scala/chisel3/experimental/package.scala

```diff
@@ -130,6 +132,35 @@ package object experimental {
     }
   }
 
+  /** This class provides the `Lit` method needed to define a `Vec` literal
+    */
+  object VecLiterals {
+    implicit class AddVecLiteralConstructor[T <: Vec[_]](x: T) {
+      /** Given a generator of a list tuples of the form [Int, Data]
+        * constructs a Vec literal, parallel concept to `BundleLiteral`
+        *
+        * @param elems generates a lit of `(Int, Data)`
+        * @return
+        */
+      def Lit(elems: (Int, Data)*): T = {
+        x._makeLit(elems: _*)
+      }
+    }
+
+    implicit class AddObjectLiteralConstructor[T <: Vec.type](x: T) {
+      /** This provides an literal construction method for cases using
+        * object `Vec` as in `Vec.Lit(1.U, 2.U)`
+        */
+      def Lit[T <: Data](elems: T*)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): Vec[T] = {
```

Comment:
```suggestion
    implicit class AddObjectLiteralConstructor(x: Vec.type) {
      /** This provides an literal construction method for cases using
        * object `Vec` as in `Vec.Lit(1.U, 2.U)`
        */
      def Lit[T <: Data](elems: T*)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): Vec[T] = {
```
No type parameter needs for `Vec.type`, it's an object with no type parameters.

Why do we have the implicits here but not on the version of the implicit for class `Vec`?